ambry-frontend/src/main/java/com.github.ambry.frontend/AmbrySecurityService.java

```diff
@@ -150,7 +150,6 @@ private Long getIfModifiedSinceMs(RestRequest restRequest) {
   private void setHeadResponseHeaders(BlobInfo blobInfo, RestResponseChannel restResponseChannel)
       throws RestServiceException {
     BlobProperties blobProperties = blobInfo.getBlobProperties();
-    restResponseChannel.setHeader(RestUtils.Headers.LAST_MODIFIED, new Date(blobProperties.getCreationTimeInMs()));
```

Comment:
Fixed the log message. Thanks for catching.

This is removed because this particular header is set even before this function is called (check the caller `processResponse()`). So this was a redundant statement.